modules/@apostrophecms/admin-bar/ui/apos/components/TheAposContextBar.vue

```diff
@@ -142,24 +142,17 @@ export default {
       sessionStorage.setItem('aposTabId', tabId);
     }
     window.apos.adminBar.tabId = tabId;
-
-    if (this.editMode) {
-      // Watch out for legacy situations where edit mode is active
-      // but we are not in draft
-      if (!await this.lock(`${this.action}/${this.context._id}`)) {
-        this.lockNotAvailable();
-        return;
+    const lastBaseContext = JSON.parse(sessionStorage.getItem('aposLastBaseContext') || '{}');
+    if (lastBaseContext.aposDocId === this.context.aposDocId) {
+      if (lastBaseContext.draftMode !== this.draftMode) {
+        await this.setContext({ mode: lastBaseContext.draftMode });
       }
-      if (this.draftMode !== 'draft') {
```

Comment:
More of the dead code for a no longer extant edge case that predates the change away from storing the editing state in the session.